gensim/models/doc2vec.py

```diff
@@ -600,18 +600,20 @@ def __init__(self, documents=None, size=300, alpha=0.025, window=8, min_count=5,
           of the model.
 
         """
+
         super(Doc2Vec, self).__init__(
-            size=size, alpha=alpha, window=window, min_count=min_count, max_vocab_size=max_vocab_size,
-            sample=sample, seed=seed, workers=workers, min_alpha=min_alpha,
-            sg=(1+dm) % 2, hs=hs, negative=negative, cbow_mean=dm_mean,
+            size=size, window=window, min_count=min_count,
+            sample=sample, workers=workers,
```

Comment:
If `size`, `window`, `min_count`, `sample`, and `workers` all use the same defaults as Word2Vec, there's no need to re-specify them in this method signature nor pass to the call of superclass `__init__()`.